app/jobs/hearings/monitor_box_job.rb

```diff
@@ -23,24 +23,29 @@ def initialize
   end
 
   def perform
-    poll_box_dot_com_for_new_files
+    files = poll_box_dot_com_for_new_files
+    files.count > 0 ? download_box_files(files) : true
   rescue StandardError => error
     log_error(error)
     raise error
   end
 
-  private
-
-  def box_parent_folder_id
-    ENV["BOX_PARENT_FOLDER_ID"]
-  end
-
   def poll_box_dot_com_for_new_files
     files = net_new_files
 
     files_with_permitted_keys(files)
   end
 
+  def download_box_files(files)
+    Hearings::VaBoxDownloadJob.perform_now(files)
```

Comment:
```suggestion
    Hearings::VaBoxDownloadJob.perform_later(files)
```
This will allow for the next job to run asynchronously and independently of the first.

Reply:
Done!